Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM --platform=x86_64 debian:bookworm AS firmware-builder
+
+# FIRMWARE_GIT should be a git repo with the firmware source code
+ARG FIRMWARE_GIT=https://github.com/flipperdevices/flipperzero-firmware.git
+
+# Install dependencies
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    ca-certificates \
+    git \
+    curl \
+    clang \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /app
+
+RUN git clone --recursive $FIRMWARE_GIT firmware
+
+WORKDIR /app/firmware
+
+RUN ./fbt
```

Comment:
This is going to build whatever happens to be in the default branch (the `dev` branch, which in particular has no guarantees of stability). It should instead take a specific tag corresponding to the firmware we want the bindings to be API-compatible with.

Reply:
adding another ARG to the Dockerfile.